template/package.json

```diff
@@ -9,9 +9,11 @@
   },
   "dependencies": {
     "core-js": "^3.3.1",
+    "eslint-config-tram-one": "^3.0.0",
     "jest": "^24.8.0",
     "jest-transform-stub": "^2.0.0",
     "parcel-bundler": "^1.6.1",
+     "parcel-plugin-nuke-dist": "^1.0.0",
```

Comment:
I know this might not be caught by the linter, but the indentation here looks off by one space. This should be aligned with the surrounding lines.

Reply:
fixed

Reply:
don't worry about that, I squash and merge all the PRs for this project 😄